### PR TITLE
Removing infra  from logging docs

### PR DESCRIPTION
**Pull request by mburke5678**
https://jira.coreos.com/browse/OSDOCS-254

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@mburke5678 labels

**Comment by mburke5678** [comment]
@anpingli Please take a look.

**Comment by mburke5678** [comment]
@anping PTAL

**Comment by mburke5678** [comment]
@anpingli PTAL

File: modules/efk-logging-eventrouter-deploy.adoc [hunk 1]
```diff
@@ -68,20 +68,20 @@ objects:
...
       labels:
         component: eventrouter
-        logging-infra: eventrouter
+        logging: eventrouter
         provider: openshift
     spec:
       selector:
         matchLabels:
           component: eventrouter
-          logging-infra: eventrouter
```
Comment (anpingli) [inline modules/efk-logging-eventrouter-deploy.adoc]: Please keep the infra in this resource file.

**Review by anpingli — COMMENTED** [review]
LGTM now

**Pull request merged**